index.js

```diff
@@ -185,7 +202,9 @@ walletRouter.post('/simple-send-tx', (req, res) => {
   const wallet = req.wallet;
   const address = req.body.address;
   const value = parseInt(req.body.value);
-  const ret = wallet.sendTransaction(address, value);
+  // Expects object with {'uid', 'name', 'symbol'}
+  const token = req.body.token || null;
```

Comment:
Use `tokenId` instead of `token`?

Reply:
I agree 100% with you, however we should have a lib refactor to support this. 

I had this discussion with @obiyankenobi here: https://github.com/HathorNetwork/hathor-wallet-lib/pull/111#discussion_r544531482

Reply:
Ok. Do we have another issue to refactor it later? A simple way is to create a new method that receives only the `tokenId` and make this method call the new one passing `token.uid`. What do you think?

Reply:
Created the issue. When we create the new method we can do exactly what you said.